packages/runner-ct/src/app/RunnerCt.tsx

```diff
@@ -238,7 +249,7 @@ const App: React.FC<AppProps> = observer(
       }
       : {}
 
-    const filteredSpecs = props.state.specs.filter((spec) => spec.relative.toLowerCase().includes(search.toLowerCase()))
+    const { searchInput, orderedResults: filteredSpecs, onSearch } = useFuzzySearch(props.state.specs, undefined, fuzzyConfig)
```

Comment:
Might be good to have a test, considering all the regressions we've been running into.

We will refactor the spec list and related logic, but the test should be able to survive a refactor. We have a test that renders `<RunnerCt />`, you could just type something in the search and see if it's working as expected.

Reply:
Oh good idea. As mentioned in the description I'm going to be adding Jest tests, but a Cypress test is good too.